backend/src/main/java/com/redhat/cloud/notifications/db/repositories/EndpointRepository.java

```diff
@@ -93,18 +93,6 @@ public List<Endpoint> getEndpointsPerCompositeType(String accountId, @Nullable S
         return endpoints;
     }
 
-    public EndpointType getEndpointTypeById(String accountId, UUID endpointId) {
-        String query = "Select e.compositeType.type from Endpoint e WHERE e.accountId = :accountId AND e.id = :endpointId";
-        try {
-            return entityManager.createQuery(query, EndpointType.class)
-                    .setParameter("accountId", accountId)
-                    .setParameter("endpointId", endpointId)
-                    .getSingleResult();
-        } catch (NoResultException e) {
-            return null;
-        }
-    }
-
```

Comment:
Does this method has a problem? or why it was removed / replaced? 

This method prevent's a `loadProperties` call. The same `endpointType == null` could be used or this could even throw a `NotFoundException`.

Reply:
Good catch, I didn't notice that the other one was loading the properties.

I'll update the PR with a better fix.